spec/integration/_dev/spec.yml

```diff
@@ -22,3 +22,8 @@ spec:
       name: test
       required: false
       $ref: "./test/spec.yml"
+    - description: Folder containing shared files.
+      type: folder
+      name: shared
+      required: false
+      $ref: "./shared/spec.yml"
```

Comment:
Should this be added also to content packages?

Reply:
I thought about that, but content packages are only supposed to hold files that we do not support links from currently IIRC